thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/dao/GenericPojoDao.java

```diff
@@ -163,8 +166,23 @@ public Connection getConnection() throws SQLException {
     return dataSource.getConnection();
   }
 
+  private String getCurrentPrincipal() {
```

Comment:
- we will need to add a mocked user session provider for unit testing. Something that we would have when implementing authorization.